filebeat/crawler/crawler.go

```diff
@@ -33,7 +33,7 @@ func New(out prospector.Outlet, prospectorConfigs []*common.Config, beatDone cha
 	}, nil
 }
 
-func (c *Crawler) Start(r *registrar.Registrar, reloaderConfig *common.Config) error {
+func (c *Crawler) Start(r *registrar.Registrar, configProspectors *common.Config) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Crawler.Start should have comment or be unexported